extensions/default/src/ViewerLayout/constants/panels.ts

```diff
@@ -0,0 +1,30 @@
+const panelGroupDefination = {
+  groupId: 'viewerLayoutResizablePanelGroup',
+  shared: {
+    expandedInsideBorderSize: 4,
+    collapsedInsideBorderSize: 4,
+    collapsedOutsideBorderSize: 8,
+    collapsedWidth: 25,
+  },
+  left: {
+    // id
+    panelId: 'viewerLayoutResizableLeftPanel',
+    // expanded width
+    expandedDefaultWidth: 282,
+    // expanded width + expanded inside border
+    expandedDefaultOffsetWidth: 145 + 4,
+    // initial expanded width
+    initialExpandedDefaultOffsetWidth: 282 + 4,
```

Comment:
We should use declared constants instead of hardcoded values. If ever the border sizes change, we will have to change multiple lines of code. Not cool.

Reply:
fixed